frontend/src/views/OrderView.vue

```diff
@@ -1,74 +1,183 @@
+  const payload: OrderCreate = orderInfo.value as OrderCreate;
 
-let decription = ref();
+  if (selectedClient.value) {
+    payload.client_id = selectedClient.value.id;
+  }
+  const order = await createOrder(payload);
+  router.push(`${ROUTES.ORDER.path}/${order.id}/edit`);
+};
+
+onMounted(async () => {
+  clientsList.value = (await getClients()).data.map((client: Client) => {
+    let full_name: string = "";
+
+    if (client.first_name) full_name += client.first_name + " ";
+
+    if (client.last_name) full_name += client.last_name;
+
+    return {
+      ...client,
+      full_name,
+    };
+  });
+});
 </script>
 <template>
   <form>
-    <div class="flex flex-col gap-y-6">
-      <p class="font-bold">Daten</p>
-      <FloatLabel>
-        <InputText id="constructionProject" class="w-full" />
-        <label for="constructionProject">Bauvorhaben</label>
-      </FloatLabel>
-      <FloatLabel>
-        <Dropdown v-model="status" :options="orderStatusTypes" class="w-full" id="select" />
-        <label for="select">Status</label>
-      </FloatLabel>
-      <p class="font-bold">Skonto</p>
-      <div class="grid grid-cols-3 gap-1 mt-3">
-        <FloatLabel>
-          <Dropdown
-            v-model="discount"
-            :options="discountChoice"
-            class="w-full"
-            id="selectDiscount"
-          />
-          <label for="selectDiscount">Skontoberechtig</label>
-        </FloatLabel>
-        <FloatLabel>
-          <Dropdown
-            v-model="discountPeriod"
-            :options="discountPeriodChoice"
-            class="w-full"
-            id="select"
-          />
-          <label for="select">Skontodauer</label>
-        </FloatLabel>
-        <FloatLabel>
-          <InputText id="percent" class="w-full" />
-          <label for="percent">Skonto(%)</label>
-        </FloatLabel>
-      </div>
-      <div>
-        <label for="description" class="w-full my-3">Beschreibung</label>
-        <Textarea v-model="decription" rows="5" class="w-full" id="description" />
-      </div>
-      <div class="my-1">
-        <p class="font-bold">Kunde</p>
-        <Button class="my-2" type="button" label="Kunden Zuordnen" severity="secondary"></Button>
-      </div>
+    <div class="flex flex-row justify-between mb-3">
+      <Button
+        @click="onOrdersList"
+        icon="pi pi-arrow-left"
+        size="small"
+        severity="secondary"
+        text
+        raised
+      />
+      <Button
+        @click="onSaveOrder"
+        type="button"
+        label="Auftrag Speichern"
+        :disabled="isButtonDisabled"
+      ></Button>
+      <Button v-if="isEditing" type="button" label="Löschen" severity="danger" text raised></Button>
     </div>
+    <Card>
+      <template #content>
+        <div class="flex flex-col gap-y-5">
+          <p class="font-bold">Daten</p>
+          <FloatLabel>
+            <InputText id="constructionProject" v-model="orderInfo.title" class="w-full" />
```

Comment:
wouldn't it make sense to keep html IDs closer to the naming of the domain?

```suggestion
            <InputText id="order-title" v-model="orderInfo.title" class="w-full" />
```